schema/rdl_schema_0.1.json

```diff
@@ -1015,6 +1023,56 @@
 				}
 			}
 		},
+		"Organization": {
+			"type": "object",
+			"description": "The organizations, consortia and individuals involved in the creation and maintanence of the dataset.",
```

Comment:
This definition describes a single organization so the description should begin 'An organization...'

Reply:
I'm not sure the parties/roles modelling is the right approach here. The main motivations for using that approach in OCDS were to reduce repetition when the same organisation appeared multiple times in a contracting process and to provide flexibility for listing the many different organisations and roles that can be involved in a contracting process. However, the downsides of the approach are increasing complexity for data publishers, users and validation tooling.

Thinking of a simple use case for RDLS of displaying the author of a dataset, it's easier to check a field like `author.name` than it is to iterate through a `parties` array checking for an item with `.role` set to `author`.

My other concern is the mixing of individuals, organisations and consortia

I think that we should look at values in the `enum` for `.role` and decide how to model each.

Reply:
Continued in https://github.com/GFDRR/rdl-standard/issues/61